packages/block-library/src/calendar/style.scss

```diff
@@ -0,0 +1,37 @@
+.wp-block-calendar {
+	text-align: center;
+
+	th,
+	tbody td {
+		padding: 4px;
+		border: 1px solid $light-gray-500;
+	}
+
+	tfoot td {
+		border: none;
+	}
+
+	table {
+		width: 100%;
+		border-collapse: collapse;
+		font-family: $default-font;
+	}
+
+	table th {
+		font-weight: 440;
+		background: $light-gray-300;
+	}
+
+	a {
+		text-decoration: underline;
+	}
+
+	tfoot a {
+		color: #0073aa;
```

Comment:
For consistency in the interface, could we re-use an existing colour in `assets/stylesheets/_colors.scss`?